writer.go

```diff
@@ -132,11 +139,12 @@ type WriterStats struct {
 	Rebalances int64 `metric:"kafka.writer.rebalance.count" type:"counter"`
 	Errors     int64 `metric:"kafka.writer.error.count"     type:"counter"`
 
-	DialTime  DurationStats `metric:"kafka.writer.dial.seconds"`
-	WriteTime DurationStats `metric:"kafka.writer.write.seconds"`
-	WaitTime  DurationStats `metric:"kafka.writer.wait.seconds"`
-	Retries   SummaryStats  `metric:"kafka.writer.retries.count"`
-	BatchSize SummaryStats  `metric:"kafka.writer.batch.size"`
+	DialTime       DurationStats `metric:"kafka.writer.dial.seconds"`
+	WriteTime      DurationStats `metric:"kafka.writer.write.seconds"`
+	WaitTime       DurationStats `metric:"kafka.writer.wait.seconds"`
+	Retries        SummaryStats  `metric:"kafka.writer.retries.count"`
+	BatchSize      SummaryStats  `metric:"kafka.writer.batch.size"`
+	BatchSizeBytes SummaryStats  `metric:"kafka.writer.batch.size.bytes"`
```

Comment:
```suggestion
	BatchBytes SummaryStats  `metric:"kafka.writer.batch.bytes"`
```
This would better follow the naming scheme of other metrics.